go.mod

```diff
@@ -25,20 +27,21 @@ require (
 	github.com/soheilhy/cmux v0.1.5
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/viper v1.9.0
-	github.com/stretchr/testify v1.7.1
+	github.com/stretchr/testify v1.7.2
 	github.com/vmihailenco/msgpack/v5 v5.3.5
 	go.uber.org/atomic v1.9.0
 	go.uber.org/zap v1.19.1
 	google.golang.org/genproto v0.0.0-20220118154757-00ab72f36ad5
 	google.golang.org/grpc v1.43.0
-	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.23.3
-	k8s.io/apimachinery v0.23.3
-	k8s.io/client-go v0.23.3
-	k8s.io/code-generator v0.23.3
-	k8s.io/gengo v0.0.0-20210813121822-485abfe95c7c
-	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
-	k8s.io/utils v0.0.0-20211116205334-6203023598ed
+	google.golang.org/protobuf v1.28.0
+	k8s.io/api v0.24.2
```

Comment:
Are these updates acceptable?

Reply:
should be good.

Reply:
hold on.... the versions are not consistent... let's use `v0.23.3`.

Reply:
I am going to close this PR and start from scratch